src/Client/Client.php

```diff
@@ -55,13 +56,24 @@ public function createRequest(Request $request)
      *
      * @param \Assertis\Http\Request\Request $request
      * @return ResponseInterface
+     * @throws Exception
      */
     public function send(Request $request)
     {
         try {
             $response = $this->guzzleClient->send($this->createRequest($request));
-        } catch (RequestException $exception) {
-            $response = $exception->getResponse();
+        } catch (Exception $exception) {
+            if ($exception instanceof RequestException) {
+                $response = $exception->getResponse();
+
+                if (empty($response)) {
+                    $url = $request->getUrl();
+                    throw new RuntimeException("Response from request $url was null.");
```

Comment:
It can happen. Mostly, its why we are doing this fix - because is happening and we are not sure why and from what reasons.